Examples/BookingSystem.AspNetCore/Feeds/OrdersFeed.cs

```diff
@@ -58,24 +58,32 @@ x.Modified < (DateTimeOffset.UtcNow - new TimeSpan(0, 0, 2)).UtcTicks)
                                         PriceCurrency = "GBP"
                                     },
                                     OrderedItem = RenderOpportunityWithOnlyId(orderItem.OpportunityJsonLdType, new Uri(orderItem.OpportunityJsonLdId)),
-                                    AccessCode = new List<PropertyValue>
+                                    AccessChannel = orderItem.MeetingUrl != null ? new VirtualLocation()
+                                    {
+                                        Name = "Zoom Video Chat",
+                                        Url = orderItem.MeetingUrl,
+                                        AccessId = orderItem.MeetingId,
+                                        AccessCode = orderItem.Password,
```

Comment:
Really very minor comment
```suggestion
                                        AccessCode = orderItem.MeetingPassword,
```

Reply:
👍